.github/workflows/python-package.yml

```diff
@@ -8,26 +8,27 @@ on:
     branches: [ master ]
     paths-ignore:
       - '**.md'    
-      - '**.rst'    
   pull_request:
-    branches: [ master ]
```

Comment:
This has restricted the CI run to PRs from the master branch itself, which is usually not what is wanted.